### PR TITLE
[by mistake merged without squash] Read anndata file as the part of package

### PR DESCRIPTION
**Pull request by siberianisaev**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,12 +19,10 @@ By default, `CapAnnData` does not automatically read any data. To begin working
 
 ```python
 import h5py
```
Comment (rm1113) [inline README.md]: please remove import

Comment (rm1113) [reply README.md]: Please also refactor snippets in sectins 4,5,6

**Review by rm1113 — APPROVED** [review]
Please refactor the documentation

File: README.md (hunk 1 above)
Comment (siberianisaev) [reply README.md]: ty, done

**Pull request merged**